### PR TITLE
8298645 JNI works with accessibleSelection on a wrong thread

### PR DESCRIPTION
**Pull request by savoptik**
[ComboBoxAccessibility accessibilityValue] works with the accessibleSelection object directly, however the work should go through CAccessibility so that it is executed on the Event Dispatch thread.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8298645](https://bugs.openjdk.org/browse/JDK-8298645): JNI works with accessibleSelection on a wrong thread

### Reviewers
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**) ⚠️ Review applies to [38319bf7](https://git.openjdk.org/jdk/pull/11653/files/38319bf724d9b43d0ab93bfff86a450f6e001297)
 * [Alexander Zuev](https://openjdk.org/census#kizune) (@azuev-java - **Reviewer**) ⚠️ Review applies to [38319bf7](https://git.openjdk.org/jdk/pull/11653/files/38319bf724d9b43d0ab93bfff86a450f6e001297)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11653/head:pull/11653` \
`$ git checkout pull/11653`

Update a local copy of the PR: \
`$ git checkout pull/11653` \
`$ git pull https://git.openjdk.org/jdk pull/11653/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11653`

View PR using the GUI difftool: \
`$ git pr show -t 11653`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11653.diff">https://git.openjdk.org/jdk/pull/11653.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/macosx/classes/sun/lwawt/macosx/CAccessibility.java [hunk 1]
```diff
@@ -817,6 +817,26 @@ public Object[] call() throws Exception {
         }, c);
     }
 
+
+    private static Accessible getAccessibleComboboxValue(Accessible a, Component c) {
```
Comment (SWinxy) [inline src/java.desktop/macosx/classes/sun/lwawt/macosx/CAccessibility.java:821]: A comment or javadoc should mention that this is called from native code, specifically pointing to `ComboBoxAccessibility.m`.

Comment (savoptik) [reply src/java.desktop/macosx/classes/sun/lwawt/macosx/CAccessibility.java:821]: Done

**Review by mrserb — APPROVED** [review]

**Comment by azuev-java** [comment]
Please update copyright years in both changed files. Other than that looks good.

**Review by azuev-java — APPROVED** [review]

**Pull request closed without merging**